### PR TITLE
Boltzmann brain experiment logic

### PR DESCRIPTION
**Pull request by BoltzmannBrain**

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@scottpurdy merging so I can keep working with these changes.
These changes are simply what you suggested earlier today.

File: fluent/models/classify_randomSDR.py [hunk 1]
```diff
@@ -46,22 +46,18 @@ def __init__(self, verbosity=1):
...
+    @param text       (str)             String to encode.
     @return           (numpy.array)     Bitmap of the SDR, as a numpy array of
                                         integers.
     """
-    state = random.getstate()
-    random.seed(string)
-    bitmap = numpy.array(random.sample(xrange(self.n), self.w), dtype=int)
-    random.setstate(state)
+    random.seed(text)
+    bitmap = numpy.array(random.sample(xrange(self.n), self.w), dtype="int8")
```
Comment (scottpurdy) [inline fluent/models/classify_randomSDR.py]: If you use `nupic.bindings.math.Random.sample` then the sampled elements will be in the same order as the originals, meaning you don't need to sort.

Comment (BoltzmannBrain) [inline fluent/models/classify_randomSDR.py]: @scottpurdy is there any documentation for `nupic.bindings.math.Random`? Maybe an example on how to seed with a string?

Comment (scottpurdy) [inline fluent/models/classify_randomSDR.py]: I don't think you can seed with a string. You could use hashlib to hash to an integer and seed with that. We may also want to add that functionality to the Random implementation if it looks generally useful.

**Pull request merged**